ext/ddtrace_profiling_native_extension/stack_recorder.c

```diff
@@ -214,6 +217,7 @@ static VALUE _native_initialize(
   VALUE cpu_time_enabled,
   VALUE alloc_samples_enabled,
   VALUE heap_samples_enabled,
+  VALUE heap_sizes_enabled,
```

Comment:
Minor: Just to keep it in-sync with what's below...

```suggestion
  VALUE heap_size_enabled,
```

...I really dislike that C forces us to do this (or otherwise order our code in really awkward ways :(((